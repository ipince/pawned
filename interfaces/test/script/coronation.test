# Coronation
StartNewGame human 0 human 0
MakeMove h2-h4 0
MakeMove c7-c6 0
MakeMove e2-e3 0
MakeMove b7-b6 0
MakeMove f1-e2 0
MakeMove h7-h5 0
MakeMove e2-h5 0
MakeMove h8-h5 0
MakeMove d1-h5 0
MakeMove g7-g6 0
MakeMove h5-g6 0
MakeMove f7-g6 0
MakeMove d2-d3 0
MakeMove g8-f6 0
MakeMove e3-e4 0
MakeMove f6-e4 0
MakeMove d3-e4 0
MakeMove g6-g5 0
MakeMove c1-g5 0
MakeMove f8-h6 0
MakeMove g5-h6 0
MakeMove a7-a6 0
MakeMove h4-h5 0
MakeMove a6-a5 0
MakeMove h6-c1 0
MakeMove a5-a4 0
MakeMove h5-h6 0
MakeMove b6-b5 0
MakeMove h6-h7 0
MakeMove b5-b4 0
MakeMove h7-h8 0