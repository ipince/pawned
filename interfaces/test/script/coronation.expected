# Coronation
New game started
h2-h4
c7-c6
e2-e3
b7-b6
f1-e2
h7-h5
e2-h5
h8-h5
d1-h5
g7-g6
h5-g6
f7-g6
d2-d3
g8-f6
e3-e4
f6-e4
d3-e4
g6-g5
c1-g5
f8-h6
g5-h6
a7-a6
h4-h5
a6-a5
h6-c1
a5-a4
h5-h6
b6-b5
h6-h7
b5-b4
h7-h8
